libs/frontend/feature-build/src/lib/use-copy-paste.ts

```diff
@@ -70,7 +65,14 @@ export const useCopyPaste = (): IUseCopyPaste => {
           pasteStyle(copiedComponent)
         }
       } else {
-        pasteComponent(copiedComponent)
+        if (!copiedComponent.parentId) {
```

Comment:
This is probably good to do, but is there anything actually blocking us from pasting the root component into an arbitrary component? Maybe I'll test it out 🤔